docs/user/introduction.md

```diff
@@ -22,7 +22,7 @@ You can [search](https://anaconda.org/) for packages online. Look out for packag
 
 ## Why conda-forge?
 
-The packaging team from [Anaconda, Inc.](https://anaconda.org/), packages a multitude of packages and provides them to all users free of charge in their `defaults` channel.
+The packaging team from [Anaconda, Inc.](https://anaconda.org/) provides a multitude of packages on their `defaults` channel.
```

Comment:
This was likely written a long time ago. Given ToS, this really shouldn't say free of change to all users

However don't want to get into all of that in an intro. So just shortened this instead

Honestly this whole section may benefit from a revamp. In interim this seems like a small reasonable change